homeassistant/components/logbook.py

```diff
@@ -133,14 +133,21 @@ def __init__(self, config):
         else:
             datetime = dt_util.start_of_local_day()
 
-        start_day = dt_util.as_utc(datetime)
-        end_day = start_day + timedelta(days=1)
+        period = request.query.get('period')
```

Comment:
What's this change about period? That does not seem related to the entity_id change